plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/AbstractKuduWithStandardInferSchemaConnectorTest.java

```diff
@@ -17,6 +17,7 @@
 
 import java.util.Optional;
 
+import static io.trino.testing.sql.TestTable.randomTableSuffix;
```

Comment:
How about changing the commit to
"Rename Kudu test and override missing test method"